docs/concepts/trading-on-vega/discounts-rewards.md

```diff
@@ -103,9 +104,9 @@ Some rewards measure trader activity over a number of epochs (set per reward). T
 ### How rewards are scaled
 Since rewards can only be provided if they're funded, the [recurring tranfer](../assets/transfers.md#recurring-transfers) that's used to fund those rewards also includes details on how the final reward amount is calculated.
 
-Pro-rata: A participant's reward is scaled based on their activity streak and/or how long they've kept previous reward earnings in their vested rewards account, which will influence how high up the rankings they are.
+**Pro-rata**: A participant's reward is scaled based on their activity streak and/or how long they've kept previous reward earnings in their vested rewards account, which will influence how high up the rankings they are.
 
-Rank: A participant's reward is scaled based on where they sit on the list of traders who are eligible for the reward. Those higher up the list receive a higher ratio of the reward for each reward period.
+**Rank**: A participant's reward is scaled based on where they sit on the list of traders who are eligible for the reward. Those higher up the list receive a higher ratio of the reward for each reward period.
```

Comment:
I think this leads me to think activity streak + reward hoarding doesn't affect reward distribution under the rank strategy which it should.

I think the ideas to get across are:
- pro-rata: share of rewards dependent on your score
- rank: share of rewards dependent on your rank (when ranked by score)
- then for both your share is scaled by multipliers from the activity streak and vesting programs.